README.rst

```diff
@@ -181,7 +181,7 @@ On Debian or Ubuntu you want these packages:
 
 ::
 
-   sudo apt install build-essential cmake libpcre2-dev gettext
+   sudo apt install build-essential cmake libpcre2-dev gettext libncurses-dev
```

Comment:
If you run `git blame` on this line you'll find commit f62f1aaf9 (Remove remaining mentions of curses, 2024-02-23).
Since an earlier neighboring commit, ncurses is no longer a dependency but terminfo is.
But actually almost no user should even need terminfo because we default to assume xterm compatibilty.
So I'm not sure if we should even recommend installing that. It be great to move away from terminfo too, to avoid issues in SSH etc. and make the behavior more predictable